botogram/objects/updates.py

```diff
@@ -24,7 +25,8 @@ class Update(BaseObject):
         "message": Message,
         "edited_message": Message,
         "channel_post": Message,
-        "edited_channel_post": Message
+        "edited_channel_post": Message,
+        "callback_query": CallbackQuery
```

Comment:
Add a trailing comma here so you don't need to change two lines the next time you want to add a value :)